engine/src/main/scala/cromwell/engine/workflow/ShadowWorkflowActor.scala

```diff
@@ -0,0 +1,19 @@
+package cromwell.engine.workflow
+
+import akka.actor.{Props, ActorLogging, Actor}
+
+object ShadowWorkflowActor {
+  def props(): Props = Props(new ShadowWorkflowActor)
+}
+
+/**
+  * Class that will [eventually] orchestrate a single workflow.
+  */
+class ShadowWorkflowActor extends Actor with ActorLogging{
```

Comment:
I know this is just the shadow's skeleton (this is starting to get kinda creepy :smile: ), but it'll be an FSM eventually right ?